### PR TITLE
fix http protocol service name bug

### PR DESCRIPTION
**Pull request by JoeCqupt**

### CODE REVIEW

**Comment by JoeCqupt** [comment]
#86

File: brpc-java-core/src/main/java/com/baidu/brpc/protocol/http/HttpRpcProtocol.java [hunk 1]
```diff
@@ -177,7 +177,7 @@ public Object decode(ChannelHandlerContext ctx, DynamicCompositeByteBuf in, bool
     @Override
     public ByteBuf encodeRequest(Request request) throws Exception {
         HttpRequest httpRequest = (HttpRequest) request;
-        String serviceName = httpRequest.getTargetMethod().getDeclaringClass().getSimpleName();
+        String serviceName = httpRequest.getTargetMethod().getDeclaringClass().getName();
```
Comment (wenweihu86) [inline brpc-java-core/src/main/java/com/baidu/brpc/protocol/http/HttpRpcProtocol.java]: if user use BrpcMeta, it should use the serviceName of BrpcMeta

Comment (JoeCqupt) [reply brpc-java-core/src/main/java/com/baidu/brpc/protocol/http/HttpRpcProtocol.java]: if user use  BrpcMeta , line 184  will set the serviceName of BrpcMeta

Comment (wenweihu86) [reply brpc-java-core/src/main/java/com/baidu/brpc/protocol/http/HttpRpcProtocol.java]: @JoeCqupt OK. I don't see this code originally.

**Pull request merged**